src/components/Hello.test.ts

```diff
@@ -1,11 +1,11 @@
-import { Hello } from './Hello';
 import { expect } from 'chai';
 import 'mocha';
+import { Hello } from './Hello';
```

Comment:
why did this get moved? was this part of the linter?

Reply:
I believe so... @katerberg Thoughts?